packages/templates/src/cli.js

```diff
@@ -0,0 +1,149 @@
+#!/usr/bin/env node
+/*
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/* eslint-disable no-await-in-loop, no-console */
+
+/**
+ * External dependencies
+ */
+import fs from 'fs';
+import puppeteer from 'puppeteer';
+
+const screenshotsPath = `build/template-posters/`;
+
+fs.mkdirSync(screenshotsPath, { recursive: true });
+
+(async () => {
+  const browser = await puppeteer.launch({
+    defaultViewport: null,
+    headless: true,
+    args: [`--window-size=1600,800`],
+  });
+  const page = await browser.newPage();
+  await page.setViewport({
+    width: 1600,
+    height: 800,
+    deviceScaleFactor: 2,
+  });
+  await page.goto(
```

Comment:
I wish we could do all of this without having to go to the Dashboard...

Imagine:

1. Importing all templates one by one
2. Create HTML output from template using `getStoryMarkup`
3. Add HTML to new page using `page.setContent()`
4. Take screenshots

Unfortunately this is not possible today because `getStoryMarkup` is not in its own package yet.

So just some wishful thinking :)

Reply:
that would be _lush_ 😻 

It's quite the loop to jump through. I wanted to go through initial feedback because this is new to me - but I'd like to give us an option to just do 1 template moving forward which will help for new additions. 

Is there a plan/ticket to make `getStoryMarkup` its own package? If there is, I can link this to that as a reminder to come back and update once it's a possibility.

Reply:
There's a rough plan, but no ticket yet. I'll create one.

Reply:
Ticket: #8778